app/views/TaxFreeAmountView.scala.html

```diff
@@ -21,111 +21,122 @@
 @import includes.taxFreeAmountSummary
 @import includes.link
 
-@this(mainTemplate: MainTemplate)
+@this(main: MainTempleteNew)
 
 @(viewModel: TaxFreeAmountViewModel, appConfig: ApplicationConfig)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
         mainHeadingText = viewModel.header,
+        showGdsVersion = true,
         preHeadingText = Messages("tai.taxCode.preHeader"))
 }
 
-@mainTemplate(
+@main(
     title = viewModel.title,
     authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = Some("full-width")){
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    backLinkUrl = Some(routes.TaxAccountSummaryController.onPageLoad.url),
+    backLinkContent = Messages("tai.back-link.upper"),
+    backLinkId = "backLink",
+){
 
-    <div class="grid-row">
+    <div class="govuk-grid-row">
 
-        <div class="grid grid-2-3">
-            <div class="inner-block">
+        <div class="govuk-grid-column-two-thirds">
+            @header
+        </div>
 
-                <div id="taxFreeAmountSummary" class="section flush--bottom">
+        <div class="govuk-grid-column-two-thirds">
 
-                    <p id="ledeMessage" class="flush--top">
-                        <span class="lede">@Messages("tai.taxFreeAmount.summarysection.taxFreeExplanation")</span>
+            <div id="taxFreeAmountSummary">
+                    <p class="govuk-body-l">
+                        @Messages("tai.taxFreeAmount.summarysection.taxFreeExplanation")
                     </p>
 
-                    <h2 class="flush--bottom">@Messages("tai.taxFreeAmount.summarysection.heading")
-                        <span class="display-block bold-large section section--narrow flush--bottom">@viewModel.annualTaxFreeAmount</span>
+                    <h2 id="xyz" class="govuk-heading-m">
```

Comment:
id xyz?

Reply:
oops

Reply:
id="xyz" can be removed Maria -  I must have put it in by mistake